scripts/exglobal_atmos_products.sh

```diff
@@ -129,7 +129,7 @@ for (( nset=1 ; nset <= downset ; nset++ )); do
 
   # Run with MPMD or serial
   if [[ "${USE_CFP:-}" = "YES" ]]; then
-    "${HOMEgfs}/ush/run_mpmd.sh" "${DATA}/poescript"
+    OMP_NUM_THREADS=1 "${HOMEgfs}/ush/run_mpmd.sh" "${DATA}/poescript"
```

Comment:
I tested this on Orion and it works well with faster runtimes.  I will note that this achieves the same effect as
`WGRIB2="${WGRIB2} -ncpu 1" "${HOMEgfs}/ush/run_mpmd.sh" "${DATA}/poescript"`
But I am satisfied with this option since there is no need to run anything threaded inside of this MPMD job.

Reply:
I too am ok with this for the time being.
Note however that in the future, `run_mpmd.sh` could be used for running threaded applications, at which point, one would need to account for the index and the thread count.
We can address that when a use case presents itself.
Thanks for this PR.